.circleci/config.yml

```diff
@@ -14,17 +14,19 @@ jobs:
               prdata=$(curl -X GET -u $GITHUB_USER_TOKEN:x-oauth-basic https://api.github.com/repos/Sunbird-Obsrv/sunbird-core-dataproducts/pulls/$CIRCLE_PR_NUMBER)
               target_branch=$(echo "${prdata}" | jq -r '.base.ref')
           fi
-          echo $target_branch
-          git clone https://github.com/Sunbird-Obsrv/sunbird-analytics-core.git
-          branchExists=$(cd sunbird-analytics-core && git ls-remote --heads origin $target_branch)
-          echo $branchExists
-          if [ -z ${branchExists} ]; then
-              latest_branch=$(cd sunbird-analytics-core && git for-each-ref --count=1 --sort=-committerdate 'refs/remotes/**/release*' --format='%(refname:short)' | head -1)
-              echo $latest_branch
-              cd sunbird-analytics-core && git checkout $latest_branch
-          else
-              cd sunbird-analytics-core && git checkout $target_branch
-          fi
+          git clone https://github.com/ocisunbird/sunbird-analytics-core.git
+          cd sunbird-analytics-core && git checkout oci-5.1.0-JAPACCOE-2839-release-5.0.0_RC2
```

Comment:
@ddevadat  This looks like a release tag is hard coded in the .circleci/config.yml. This doesn't look right.

Reply:
@anandp504 you are right..lets park this PR as this needs some special attention.

The issue here is circle ci test runs. We have upgraded the cloud-store-sdk version in the analytics core repo, those changes are yet to merged in the release-5.1.1

Till that happens , everytime circle ci runs for core-dataproducts it will try to build from release-5.1.1 which doesnt have cloud-store-sdk:1.4.6 yet and will fail.

Reply:
@ddevadat  - Did we merge the analytics-core with the latest SDK changes? If yes, can we revert these changes back?

Reply:
reverted the changes as the analytics core pr is merged